install.sh

```diff
@@ -1,56 +1,59 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
 set -eu
 
-
 # Parsing arguments
 if [ -t 0 ] ; then
   printf "Micromamba binary folder? [~/.local/bin] "
   read BIN_FOLDER
-  printf "Prefix location? [~/micromamba] "
-  read PREFIXLOCATION
   printf "Init shell? [Y/n] "
   read INIT_YES
   printf "Configure conda-forge? [Y/n] "
   read CONDA_FORGE_YES
 fi
 
-
 # Fallbacks
 BIN_FOLDER="${BIN_FOLDER:-${HOME}/.local/bin}"
-PREFIXLOCATION="${PREFIXLOCATION:-${HOME}/micromamba}"
 INIT_YES="${INIT_YES:-yes}"
 CONDA_FORGE_YES="${CONDA_FORGE_YES:-no}"
 
+# Prefix location is relevant only if we want to call `micromamba shell init`
+case "$INIT_YES" in
+  y|Y|yes)
+    if [ -t 0 ]; then
+      printf "Prefix location? [~/micromamba] "
+      read PREFIX_LOCATION
+    fi
+    ;;
+esac
+PREFIX_LOCATION="${PREFIX_LOCATION:-${HOME}/micromamba}"
```

Comment:
What does this improve compared to the previous situation where user could set `INIT_YES` as an environment variable from outside of the script?

Reply:
If the user has set `INIT_YES=no` or answered `no` to the `Init shell` question, there is simply no point in asking for the prefix location, as it is not going to be used in any way.

Reply:
Ha yes, I missed that.